taipy/gui/_page.py

```diff
@@ -21,7 +21,7 @@
     from ._renderers import Page
     from .gui import Gui
 
-_DETECT_CLOSING_TAGS = re.compile(r"<([\w_\-]+)(([^>\"]+\"[^\"]*\")*\s*)?(><\/\1>)", flags=re.MULTILINE)
+_DETECT_CLOSING_TAGS = re.compile(r"<([A-Z][\w_\-]*)(([^>\"]+\"[^\"]*\")*\s*)?(><\/\1>)", flags=re.MULTILINE)
```

Comment:
Not too sure about this regexp:
`[A-Z]` forces a capital letter as the first letter for the tag... Sure about that?
`\w` already contains `_`... so `[\w_\-]` should be `[-\w]` (special meaning with `-` also)